### DISCUSSION TITLE
[@types/spotify-api] CurrentlyPlayingObject actions property

### DISCUSSION THREAD

**Discussion by bkeys818** [DefinitelyTyped/DefinitelyTyped#55172] [Issues with a @types package]
## Issue

In the [@types/spotify-api](https://www.npmjs.com/package/@types/spotify-api) package, the type `CurrentlyPlayingObject` has a property called `actions`. This is inaccurate to Spotify's documentation.

It's worth noting that there is another type called `CurrentlyPlayingContextObject`. This type is not in the package (not sure why). It extends `CurrentlyPlayingObject` and does have the property `actions`, but `CurrentlyPlayingObject` does not.
 
### Links to types
- [`CurrentlyPlayingObject` in source file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d8e284207b00681892fe2fb86ef4f2e2cdfb2d72/types/spotify-api/index.d.ts#L1575)
- [`CurrentlyPlayingObject` in Spotify's Documentation](https://developer.spotify.com/documentation/web-api/reference/#object-currentlyplayingobject)
- [`CurrentlyPlayingContextObject ` in Spotify's Documentation](https://developer.spotify.com/documentation/web-api/reference/#object-currentlyplayingcontextobject)

[a comment by an automated account is not shown]

**Comment by piksel** [comment]
Yeah, I think they changed their API to the current version of `CurrentlyPlayingObject` and renamed the old one `CurrentlyPlayingContextObject`.

It doesn't seem like anything returns `CurrentlyPlayingContextObject` in the API anymore AFAICT.

**Reply by piksel** [reply]
It also seems like the missing `device` property on the `CurrentlyPlayingObject` is present on the return type of `GET /v1/me/player`: 
https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-information-about-the-users-current-playback
That object does not have a definition in the `Objects Index` section...
